kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_Message.kt

```diff
@@ -1,27 +1,164 @@
+ * @param emoji Emojis that will be removed to the message
+ * @throws [RestRequestException] if something went wrong during the request.
+ */
+public suspend inline fun MessageBehavior.deleteReaction(userId: Snowflake, emoji: String): Unit =
+    deleteReaction(userId, emoji.toReaction())
+
+/**
+ * Requests to delete an [emoji] to this message.
+ * @param emoji Emoji that will be deleted to the message
+ * @throws [RestRequestException] if something went wrong during the request.
+ */
+public suspend inline fun MessageBehavior.deleteReaction(emoji: GuildEmoji): Unit =
+    this.deleteReaction(emoji.toReaction())
+
+/**
+ * Requests to delete an emoji with unicode format to this message.
+ * @param unicode Emoji that will be deleted to the message
+ * @throws [RestRequestException] if something went wrong during the request.
+ */
+public suspend inline fun MessageBehavior.deleteReaction(unicode: String): Unit =
+    this.deleteReaction(unicode.toReaction())
+
+/**
+ * Requests to remove an [emoji] from the own bot to this message.
+ * @param emoji Emoji that will be removed to the message
+ * @throws [RestRequestException] if something went wrong during the request.
+ */
+public suspend inline fun MessageBehavior.deleteOwnReaction(emoji: GuildEmoji): Unit =
+    deleteOwnReaction(emoji.toReaction())
+
+/**
+ * Requests to remove emoji with unicode format from the own bot to this message.
+ * @param unicode Emoji that will be removed to the message
+ * @throws [RestRequestException] if something went wrong during the request.
+ */
+public suspend inline fun MessageBehavior.deleteOwnReaction(unicode: String): Unit =
+    deleteOwnReaction(unicode.toReaction())
+
+/**
+ * Create listener for an events about the message.
+ * @receiver Message that will be listen to apply actions according to the events
+ * @param timeout Time to stop listening to events after the last received event corresponding to the message
+ * @param manage Block of code to create listeners
+ * @return The instance of manager created to manager the events
+ * @throws IllegalStateException Exception if this is impossible to start the listening of event
+ * because he is already started
+ */
+@Throws(IllegalStateException::class)
+public inline fun MessageBehavior.events(
+    timeout: Long? = MessageEventManager.DEFAULT_TIME_LISTENING,
+    manage: MessageEventManager.() -> Unit
+): MessageEventManager = MessageEventManager(this, timeout).apply {
+    manage(this)
+    if (!start()) {
+        error("Impossible to start the listening of events")
+    }
+}
+
 /** ID of the message author. **/
-public val MessageData.authorId: Snowflake get() = author.id
+public val MessageData.authorId: Snowflake
+    get() = author.id
 
 /** Is the message author a bot. **/
-public val MessageData.authorIsBot: Boolean get() = author.bot.discordBoolean
+public val MessageData.authorIsBot: Boolean
+    get() = author.bot.discordBoolean
```

Comment:
These functions fit on one line, so there was no real need to drop them to a new line.

Reply:
So maybe you have a big screen, but trust me that for small screens (like mine) having it on the next row allows you to instantly see which method is called

Reply:
My margin is set to 120 chars — are you using an 80-char TTY? As [Torvalds himself says](https://www.theregister.com/2020/06/01/linux_5_7/):

> People with restrictive hardware shouldn't make it more inconvenient for people who have better resources. Yes, we'll accommodate things to within reasonable limits. But no, 80-column terminals in 2020 isn't "reasonable" any more as far as I'm concerned. People commonly used 132-column terminals even back in the '80s, for chrissake, don't try to make 80 columns some immovable standard.

Reply:
Indeed, I had not seen that Intellij put the margin in the editor
So, it's corrected

Reply:
You may have forgotten to push that one. But I'm starting to think a lot of these stylistic changes are better handled with a more specific linter configuration, so you can leave it for now.